src/gui/integration/integration_test.go

```diff
@@ -1627,10 +1627,10 @@ func TestLiveAddressTransactions(t *testing.T) {
 		},
 	}
 
-	testAddressTransactions(t, cases)
+	testLiveAddressTransactions(t, cases)
 }
 
-func testAddressTransactions(t *testing.T, cases []addressTransactionsTestCase) {
+func testStableAddressTransactions(t *testing.T, cases []addressTransactionsTestCase) {
```

Comment:
you can move these methods back into the main test method, doesn't need to be separate methods anymore

Reply:
The `height` transaction `status` in live test is not stable.

Reply:
i mean put `testStableAddressTransactions` in `TestStableAddressTransactions` and `testLiveAddressTransactions` in `TestLiveAddressTransactions`

Reply:
oh, ok